src/backend/joanie/payment/backends/base.py

```diff
@@ -31,7 +31,7 @@ def __init__(self, configuration=None):
         self.configuration = configuration
 
     @classmethod
-    def _do_on_payment_success(cls, order, payment):
+    def _do_on_payment_success(cls, order, payment, validate_order=True):
```

Comment:
You introduce this new parameter by I don't see where you are using it with the `False` value. Am I missing something ?

Reply:
Nice catch. As I was testing, I forgot to fix the test. See https://github.com/openfun/joanie/pull/766/commits/8a8e5c0d8687c1f645580fd98cc9173a28476fd5